### PR TITLE
Update and shuffle around piping docs/links in F# symbol and operatr reference

### PR DESCRIPTION
**Pull request by cartermp**
#### Preview

✔️ [Symbol and operator reference](https://review.docs.microsoft.com/en-us/dotnet/fsharp/language-reference/symbol-and-operator-reference/?branch=pr-en-us-20113)

### CODE REVIEW

**Review by IEvangelist — APPROVED** [review]
I manually verified the links. These are false negatives with the linter, as they are not actually bare URLs.

Looks good to me, we'll :shipit: - thank you 🙏 

/cc @BillWagner - I'll let you merge this one, but it looks correct to me.

**Comment by cartermp** [comment]
Is there a ruleset or something that can be used so that future CI runs don't go 💥 ?

**Comment by IEvangelist** [comment]
> Is there a ruleset or something that can be used so that future CI runs don't go 💥 ?

It seems like you're using GitHub as your editor, since the branches are named `patch-*`. Within Visual Studio Code, we rely on the Docs Authoring Pack and Markdown linter to catch things like this. However, this seems like a bug in the linter as these are not actually bare URLs.

**Comment by BillWagner** [comment]
Hmmm. This looks like there's an error in Markdown lint. It shouldn't be triggering the bar URL warning. I'll :shipit:

**Comment by Youssef1313** [comment]
@BillWagner I think markdownlint error is correct. The link won't show correctly.

File: docs/fsharp/language-reference/symbol-and-operator-reference/index.md [hunk 1]
```diff
@@ -70,9 +67,12 @@ The following table describes symbols used in the F# language and provides a bri
 |`<>?`|[Nullable Operators](nullable-operators.md)|<ul><li>Computes the "not equal" operation when the right side is a nullable type.<br /></li></ul>|
 |`<=`|[Arithmetic Operators](arithmetic-operators.md)|<ul><li>Returns `true` if the left side is less than or equal to the right side; otherwise, returns `false`.<br /></li></ul>|
 |`<=?`|[Nullable Operators](nullable-operators.md)|<ul><li>Computes the "less than or equal to" operation when the right side is a nullable type.<br /></li></ul>|
+|<code>&#124;></code>|[Functions](../functions/index.md)|<ul><li>Passes the result of the left side to the function on the right side (forward pipe operator).<br /></li></ul>|
+|<code>&#124;&#124;></code>|[&#40; &#124;&#124;&#62; &#41;&#60;'T1,'T2,'U&#62; Function](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-core-operators.html#(%20||%3E%20))|<ul><li>Passes the tuple of two arguments on the left side to the function on the right side.<br /></li></ul>|
```
Comment (Youssef1313) [inline docs/fsharp/language-reference/symbol-and-operator-reference/index.md:71]: The problem here is `||` in the URL. Markdown parsers aren't able to detect these as part of the link and they see it as ending the current table cell.

**Pull request merged**